### PR TITLE
Added colored menu icons

### PR DESCRIPTION
**Pull request by neoatomic**
- Replaced All menu icons with colored versions.
- Created one singe sprite.
- Added new icons.
- Added loading16-light.gif as it was missing on the light theme.
- Moved anidb24.png to the correct notifiers folder
- Removed/Disabled some duplicate/unused css code. 

naamloos

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by duramato** [comment]
SS?

**Comment by neoatomic** [comment]
Another one. The add new shows page. : 

 Added colored logos to the add-show page.
- Changed the logos to colored versions.
- Created a singe sprite file.
- Removed the old black/white logos.
- Removed unnecessary code form dark.css

**Comment by OmgImAlexis** [comment]
Oh finally SickRage isn't going to look so dull anymore. :+1:

**Comment by labrys** [comment]
@neoatomic Something needs to be done about the tv logo tho.  The contrast against the light theme works well.  The antennae get lost in the dark theme, making it look sub-optimal.

File: gui/slick/css/style.css [hunk 1]
```diff
@@ -20,10 +20,10 @@ inc_top.mako
...
 }
 
-.dropdown-menu li > a:hover > [class^="menu-icon-"],
-.dropdown-menu li > a:hover > [class*=" menu-icon-"] {
-    background-image: url("../images/menu/menu-icons-white.png");
-}
+#.dropdown-menu li > a:hover > [class^="menu-icon-"],
+#.dropdown-menu li > a:hover > [class*=" menu-icon-"] {
+#    background-image: url("../images/menu/menu-icons-white-.png");
+#}
```
Comment (miigotu) [inline gui/slick/css/style.css]: `/* This is a single-line comment */`

File: gui/slick/css/style.css [hunk 2]
```diff
@@ -219,17 +219,17 @@ inc_top.mako
...
+#    [class^="menu-icon-"], [class*=" menu-icon-"] {
+#        background: url("../images/menu/menu-icons-white.png");
+#       height: 16px;
+#        width: 16px;
+#        display: inline-block;
+#        position: relative;
+#        top: 2px;
+#        float: left;
+#    }
+#}
```
Comment (miigotu) [inline gui/slick/css/style.css]: /\* This is a single-line comment */

File: gui/slick/css/style.css (hunk 1 above)
Comment (OmgImAlexis) [inline gui/slick/css/style.css]: If this is being taken out actually remove it. No need to just comment it out.

File: gui/slick/css/style.css (hunk 2 above)
Comment (OmgImAlexis) [inline gui/slick/css/style.css]: Same for here.

**Comment by neoatomic** [comment]
There where a lot of sections disabled, so i thought to follow that example. Will remove it on the next PR.
By moving to colored icons, the need for separate black/white stuff isn't needed anymore. Hopefully i will be able to remove even more.
(There are also some unused image files that i like to remove, but will bring that up when we finish this pr.)

I have the notifications section almost ready, (sprite etc.) but still needs a few tweaks, as some logos are to close to the text. Tips.?
Also made a sprite for the 16x icons on a shows page, so that will be next, but that might be a little harder to accomplish...

Hopefully at the end we have some more color in SR and all into sprite files to improve speed.

**Comment by neoatomic** [comment]
When possible i would prefer a merge to dev at this point.?

**Comment by miigotu** [comment]
It cannot be merged to dev until it builds.

**Comment by labrys** [comment]
Approved!

[Approved with PullApprove](https://pullapprove.com/SickRage/SickRage/pull-request/918/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=SickRage/SickRage)

**Pull request merged**